internal/cmd/cmd_test.go

```diff
@@ -229,3 +231,39 @@ func newTestClientConfig(srv *httptest.Server, identity api.Identity) ClientConf
 		},
 	}
 }
+
+func TestConnectorCmd(t *testing.T) {
+	var actual connector.Options
+	patchRunConnector(t, func(ctx context.Context, options connector.Options) error {
+		actual = options
+		return nil
+	})
+
+	content := `
+name: the-name
+accessKey: file:/var/run/secrets/key
+server: the-server
```

Comment:
I'll include more fields, and maybe fix in some flags as well in more tests cases, as a follow up.